core/variables.tf

```diff
@@ -41,3 +41,15 @@ variable "rds_password" {
   type        = string
   sensitive   = true
 }
+
+variable "app_port" {
```

Comment:
it can be locals variable

Reply:
Nice one, updated in [b2c2c95](https://github.com/nvminhtue/tyson_devops_ic/pull/29/commits/b2c2c950304a2bc9183d5fee716476b3672bbe34)